### DISCUSSION TITLE
Fingerprint command not supported

### DISCUSSION THREAD

**Discussion by GK-774** [jcorporation/myMPD#614] [Q&A] [answered]
libchromaprint and fpcalc works just fine from the command line, but myMPD says
"Fingerprint command not supported" when I click Song Details / Fingerprint / calculate
(and the spinning circle stays spinning, instead of stopping).

What would be needed to enable it (or if not feasible remove the option displayed)?
I did not find anything about it in the documentation, maybe someone can point me to the relevant part, if I missed it.

The log part (loglevel 7) when issuing the fingerprint command is attached.

[log_7_fingerprint.txt](https://github.com/jcorporation/myMPD/files/7576149/log_7_fingerprint.txt)

**Comment by jcorporation** [comment, marked as answer]
It is a bug in the current stable version of myMPD and already fixed in the devel branch.

**Reply by GK-774** [reply]
Well, will the (supposedly) stable version receive the bug fix as well, or is there a possibility to remove options that are not working? Im in the process of preparing two small music servers for friends and I´m unsure which of the stable versions would be the one to choose.
When I tried the devel version a few days ago it did not load at all, so I dont think thats an option I can recommend to people that dont even know what a debug log file is. Are there any improvements to the search function in devel which would reward waiting for it?

**Comment by jcorporation** [comment]
The devel branch will be the next non-patch version 8.2.0. There are many enhancements included. You can look at the milestone to decide it is worth waiting for it: https://github.com/jcorporation/myMPD/milestone/53

I actually do not plan another patch release for the v8.1.x version series.

Devel should be pretty stable and works in my environment without problems, besides of the issues noted in #579.

It would be great if you can test the devel branch again and report any found problems.
